src/install/python/python-options/python-agent-non-web-apps.mdx

```diff
@@ -0,0 +1,181 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review the manual initialization steps for non-web apps
+---
+
+It sounds like you want to monitor a non-web application, which requires a manual initialization and instrumention process.
+
+The manual process involves importing a Python agent package into your app and making a call to initialize the agent. This call modifies your app's import mechanism so that when libraries are imported, the agent listens for the function classes it recognizes.
+
+With this manual process, you can monitor arbitrary job-based systems, standalone scripts, worker processes, and non-web transactions using the `background_task` method. These transactions show up as [non-web transactions](/docs/apm/applications-menu/monitoring/transactions-page#tx_types) in the New Relic UI.
+
+The Python agent also has built-in instrumentation for the task-queuing systems [Celery](/docs/agents/python-agent/back-end-services/python-agent-celery) and Gearman.
```

Comment:
Might also be good to call out our message broker support for kafka and pika on rabbit MQ.